js-packages/@prestojs/rest/src/CursorPaginator.ts

```diff
@@ -190,4 +190,11 @@ export default class CursorPaginator extends Paginator<
             this.setCurrentState({ ...this.currentState, pageSize });
         }
     }
+
+    /**
+     * Returns true if there's more results after the current page
+     */
+    hasNextPage(): boolean {
```

Comment:
In our current implementations it's possible to know - LimitOffset and PageNumber both know the total number of records & Cursor knows the next cursor

Reply:
What were you using for the cursor implementation?

(comparison: for a python iterator you can't know whether there are more results to fetch without actually trying to do the fetch and then getting a StopIteration exception)

Reply:
DRF 

eg. last page looks like:

![image](https://user-images.githubusercontent.com/1896376/83377314-8bcf2400-a418-11ea-8cbd-db3e0e7efef5.png)

Reply:
Am not convinced that every cursor is going to be able to provide that, but if DRF does then I'm happy with it for now

Reply:
yeah it doesn't really matter - if they don't support it it can just return `true` always

it's just useful for UI to know when to show a 'more' button (so worst case is it just shows the button always)